doc/nrf/gs_installing.rst

```diff
@@ -467,6 +467,14 @@ To install |SES| Nordic Edition, complete the following steps:
     #. Copy-paste the license key and click :guilabel:`Install License`.
        The license activation window will close and SES will open the Project Explorer window.
 
+.. note::
+    Notifications about newer versions of SES are disabled in SES Nordic Edition.
```

Comment:
Is it the SES Nordic Edition or without the definite article? We have a few different version in this PR.

Reply:
I'll stick to "the" as per website: https://www.nordicsemi.com/Software-and-tools/Development-Tools/Segger-Embedded-Studio

Reply:
So, SEGGER Embedded Studio, but the SEGGER Embedded Studio Nordic Edition.